### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by yorinasub17**
This updates the README to be in line with our new format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,145 +1,123 @@
...
+### Repo organization
 
+* [modules](https://github.com/hashicorp/terraform-google-nomad/tree/master/modules): the main implementation code for this repo, broken down into multiple standalone, orthogonal submodules.
+* [examples](https://github.com/hashicorp/terraform-google-nomad/tree/master/examples): This folder contains working examples of how to use the submodules.
+* [test](https://github.com/hashicorp/terraform-google-nomad/tree/master/test): Automated tests for the modules and examples.
+* [root](https://github.com/hashicorp/terraform-google-nomad/tree/master): The root folder is *an example* of how to use the [nomad-cluster module](https://github.com/hashicorp/terraform-google-nomad/tree/master/modules/nomad-cluster) module to deploy a [Nomad](https://www.nomadproject.io/) cluster in [AWS](https://aws.amazon.com/). The Terraform Registry requires the root of every repo to contain Terraform code, so we've put one of the examples there. This example is great for learning and experimenting, but for production use, please use the underlying modules in the [modules folder](https://github.com/hashicorp/terraform-google-nomad/tree/master/modules) directly.
```
Comment (robmorgan) [inline README.md]:
```suggestion
* [root](https://github.com/hashicorp/terraform-google-nomad/tree/master): The root folder is *an example* of how to use the [nomad-cluster module](https://github.com/hashicorp/terraform-google-nomad/tree/master/modules/nomad-cluster) module to deploy a [Nomad](https://www.nomadproject.io/) cluster in [GCP](https://cloud.google.com/). The Terraform Registry requires the root of every repo to contain Terraform code, so we've put one of the examples there. This example is great for learning and experimenting, but for production use, please use the underlying modules in the [modules folder](https://github.com/hashicorp/terraform-google-nomad/tree/master/modules) directly.
```

**Review by robmorgan — CHANGES_REQUESTED** [review]
minor thing

File: README.md (hunk 1 above)
Comment (yorinasub17) [reply README.md]: Good catch!

**Comment by yorinasub17** [comment]
FYI I don't have the permissions to merge this :)

**Comment by yorinasub17** [comment]
Closing as outdated

**Pull request closed without merging**